addon/routes/gis-search-form.js

```diff
@@ -0,0 +1,36 @@
+import Ember from 'ember';
+import { Query } from 'ember-flexberry-data';
+
+export default Ember.Route.extend({
+  _metadatas: [],
+  _maps: [],
+
+  model() {
+    return { metadata: this._metadatas, maps: this._maps };
+  },
+
+  //setupController
+  actions: {
+    loadData(keyWords) {
+      let that = this;
+
+      let b1 = new Query.Builder(this.store)
+        .from('new-platform-flexberry-g-i-s-layer-metadata')
+        .selectByProjection('LayerMetadataL');
+      let getMetadata = this.store.query('new-platform-flexberry-g-i-s-layer-metadata', b1.build());
+
+      let b2 = new Query.Builder(this.store)
+        .from('new-platform-flexberry-g-i-s-map')
+        .selectByProjection('AuditView');
```

Comment:
Why exactly AuditView?

Reply:
This feature is the part of whole task, so AuditView is just as example..